include/gpiod.h

```diff
@@ -918,6 +918,28 @@ struct gpiod_line *
 gpiod_chip_get_line(struct gpiod_chip *chip, unsigned int offset) GPIOD_API;
 
 /**
+ * @brief Helper structure for storing a set of GPIO chip bulk objects.
+ */
+struct gpiod_chip_bulk {
+	struct gpiod_chip **chips;
+	/**< Dynamically allocated array of chip pointers. */
```

Comment:
The expression `dynamically allocated` is an implementation detail - please remove it.